### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by mrostecki**
This is the initial implementation of bazel-workspaces-python tool which
generates Bazel repositories with BUILD and WORKSPACE file for already
installed Python libraries.

Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>

### CODE REVIEW

File: bazel_workspaces_python/cli.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+r"""Module which is used as the bazel-workspaces-python console script.
```
Comment (manuelbuil) [inline bazel_workspaces_python/cli.py]: is that r a typo?

File: bazel_workspaces_python/template.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+"""Module which generates contents of BUILD and WORKSPACE files for Bazel."""
+
+import string
+import typing
+
+
+def generate_build_template(name: str, sources: typing.List[str]) -> str:
```
Comment (manuelbuil) [inline bazel_workspaces_python/template.py]: Seeing types being defined in python looks weird to me, but this is just a personal opinion ;)

File: bazel_workspaces_python/cli.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Attributes:
+    DEFAULT_MAIN_DIR (string): Default main path to create Bazel workspaces in.
+
+"""
+
+import argparse
+import os
+
+from bazel_workspaces_python import (
```
Comment (manuelbuil) [inline bazel_workspaces_python/cli.py]: My personal opinion is that:
`from bazel_workspaces_python import sources, template` 

is more pythonic. What you wrote looks like golang but as far as I know, there is not a PEP that sets how to do multiple imports, so this is just my personal taste

File: bazel_workspaces_python/cli.py (hunk 1 above)
Comment (mrostecki) [reply bazel_workspaces_python/cli.py]: No. It's a "raw string" in Python which doesn't treat `\` as an escape character. That's the only reason I used it here.

File: bazel_workspaces_python/template.py (hunk 2 above)
Comment (mrostecki) [reply bazel_workspaces_python/template.py]: Then we disagree. ;) I really like the type hinting in Python and I share the opinion that not using types is more error prone.

File: bazel_workspaces_python/cli.py (hunk 3 above)
Comment (mrostecki) [reply bazel_workspaces_python/cli.py]: OK, I will change it.

**Pull request merged**